disnake/activity.py

```diff
@@ -518,7 +514,7 @@ def twitch_name(self):
 
     def to_dict(self) -> Dict[str, Any]:
         ret: Dict[str, Any] = {
-            "type": ActivityType.streaming.value,
+            "type": ActivityType.streaming,
```

Comment:
are these changes safe? I know we were getting errors elsewhere from pyright and these are typed as Any, so we don't get errors here. Have they been tested?

Reply:
I was contemplating rolling them back just to be safe. I've had the changes on my own bot for a while and they worked fine, but I haven't explicitly tested each individual change-- hence the rollback. Do you/shift have any preference on this?

Reply:
probably worth a rollback, and we'll do that *all* in a separate pr

Reply:
Yup, that's what I was thinking too ^^

Reply:
Alright, should be reverted as part of 7c0fd82abfd2f9b57e63f18e4153581755ae1f92.